docs/configuration.md

```diff
@@ -223,6 +237,31 @@ log:
     </td>
 </table>
 
+### hooks
```

Comment:
Thanks @xiekeyang for the contribution.  A few things:

- The file is inconsistent whether it shows the sample and then the discussion. Let's start here fixing that and put the sample first.
- should be a H2 not H3
- we try not to leave TODO in the doc (there are some in the specs)
- "sequent type of handler" do you mean a "sequence handler"? I couldn't find many references to sequent handler so I assume you meant sequence

---> https://gist.github.com/moxiegirl/a4ea1cd782e058bb9a0d

## hooks

```yaml 
hooks:
  - type: mail
    disabled: false
    levels:
      - panic
    options:
      smtp:
        addr: smtp.sendhost.com:25
        username: sendername
        password: password
        insecure: true
      from: name@sendhost.com
      to:
        - name@receivehost.com
```

The `hooks` subsection configures the logger's handling behavior. Set 
`disabled` to `false` to configure a sequence handler such as for sending mail.  Refer
to [loglevel](#loglevel) to configure the level of messages printed.